app/src/main/java/org/kiwix/kiwixmobile/help/HelpActivity.java

```diff
@@ -65,14 +66,21 @@ protected void onCreate(Bundle savedInstanceState) {
 
   @OnClick({ R.id.activity_help_feedback_text_view, R.id.activity_help_feedback_image_view })
   void sendFeedback() {
+
+    String[] addresses = { CONTACT_EMAIL_ADDRESS };
```

Comment:
This reference is only used once so pass this array directly to the method instead of making a new reference.